warehouse/integrations/bigquery/bigquery.go

```diff
@@ -33,13 +35,13 @@ import (
 )
 
 type BigQuery struct {
-	db         *bigquery.Client
-	middleware *middleware.Client
```

Comment:
During connect itself we are setting it, to avoid haveing 2 clients: `db` and `middleware`. Also, we don't need the `getMiddleware() *middleware.Cli`